packages/dialtone-vue2/recipes/conversation_view/message_input/message_input.vue

```diff
@@ -161,7 +167,7 @@
         <dt-button
           v-if="showCancel"
           data-qa="dt-message-input-cancel-button"
-          class="dt-message-input__cancel-button"
+          class="dt-message-input__cancel-button dt-message-input__button d-p8"
```

Comment:
Add the padding on the `dt-message-input__cancel-button` class using the appropriate token.

```suggestion
          class="dt-message-input__cancel-button dt-message-input__button"
```